metrics/metrics.go

```diff
@@ -65,6 +65,18 @@ var (
 		Help: "Duration between time round received and time round expected.",
 	})
 
+	// ClientHTTPHeartbeatLatency measures the latency of HTTP hearbeat andomness requests.
+	ClientHTTPHeartbeatLatency = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "client_http_heartbeat_latency",
+		Help: "Duration between time round received and time round expected, measured on heartbeats.",
+	})
+
+	// ClientHTTPHeartbeatFailure measures the number of times HTTP heartbeats fail.
+	ClientHTTPHeartbeatFailure = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Make a `Counter`, since it only increases, and consider if it's worth having it be an overall heartbeat counter, where there's a label for distinguishing success vs failure, but one of the two increments. Having the counterpart (successes) tracked as well is useful to be able to e.g. look at % failure.

Reply:
Sounds good.